modules/cardano-node-prod.nix

```diff
@@ -14,6 +14,9 @@ with (import ./../lib.nix);
 
   deployment.ec2.elasticIPv4 = resources.elasticIPs.${"nodeip" + toString config.services.cardano-node.testIndex};
 
+  deployment.route53.accessKeyId = resources.ec2KeyPairs.${keypairFor region}.accessKeyId;
+  deployment.route53.hostName = "cardano-node-${toString config.services.cardano-node.testIndex}.aws.iohkdev.io";
```

Comment:
`iohkdev.io` should be used for `cardano-node-staging.nix`, `iohk.io` for `cardano-node-prod.nix`.